hotel-reservation-system/src/components/Hotel/HotelTable.js

```diff
@@ -245,7 +245,14 @@ export default function HotelTable() {
       setCurrentAscending("desc");
       ascending = "desc";
     }
-    loadHotels(undefined, undefined, undefined, sortField, ascending);
+    loadHotels(
```

Comment:
Are you sure that function is correct if you need to pass so many undefined values?